### DISCUSSION TITLE
Ability to set selection-foreground = cell-foreground

### DISCUSSION THREAD

**Discussion by wzxu** [ghostty-org/ghostty#2656] [Feature Requests, Ideas] [answered]
Effectively, keep the foreground text color unchanged even when selected.

Pros:
- Visually nice
- Works like how other native apps work, such as editors, browsers, and also the built-in Terminal.app
- Alacritty also already supports setting [colors.selection] text = "CellForeground", so this would be a nice feature parity.

**Comment by mitchellh** [comment]
Is the selection background in this case for say Alacritty just the window foreground?

**Reply by wzxu** [reply]
No you can set a color as well. For me I just set a dimmed blue color so it looks somewhat native.
![891](https://github.com/user-attachments/assets/2c6c5635-467f-45fd-97af-bf325e308d5c)

**Comment by mitchellh** [comment, marked as answer]
#2685
